modules/appeals_api/config/routes.rb

```diff
@@ -76,6 +76,7 @@
     namespace(:v0) { resources :api, only: [:index] }
     namespace :v1, defaults: { format: 'json' } do
       get 'decision_reviews', to: 'docs#decision_reviews'
+      get 'appeals', to: 'docs#appeals_status'
```

Comment:
This is slightly different from the path for v0, which is `/services/appeals/docs/v0/api`. This path (`/services/appeals/docs/v1/appeals`) seemed better alongside the existing `/services/appeals/docs/v1/decision_reviews` route